src/wandb_osh/lightning_hooks.py

```diff
@@ -41,6 +43,7 @@ def __init__(
         """
         super().__init__()
         self._hook = TriggerWandbSyncHook(communication_dir=communication_dir)
+        self.frequency = frequency
```

Comment:
could you make this private? i.e., `self._frequency`

Reply:
Yes, good idea